test/appium/views/transactions_view.py

```diff
@@ -39,7 +57,10 @@ def navigate(self):
             return self.TransactionDetailsView(self.driver)
 
     def get_transaction_element(self, amount: str):
-        return self.TransactionElement(self.driver, amount=amount)
+        return self.TransactionElement(self.driver, selector={'amount': amount})
+
+    def get_first_transaction(self):
+        return self.TransactionElement(self.driver, selector={'index': 0})
```

Comment:
would be better to reimplement in the same way as `element_by_text` is implemented
```
element = self.element_types["text"](self.driver)
element.locator = element.Locator.xpath_selector('//android.view.ViewGroup[@content-desc="transaction-item"]')
return element.find_elements()[0]
```